.github/workflows/linter.yml

```diff
@@ -61,4 +61,5 @@
               VALIDATE_GITLEAKS: false
               VALIDATE_JSCPD: false
               VALIDATE_JAVA: false
-              VALIDATE_GOOGLE_JAVA_FORMAT: false
\ No newline at end of file
+              VALIDATE_GOOGLE_JAVA_FORMAT: false
+              VALIDATE_CPP: false
```

Comment:
Do you need to disable `VALIDATE_CLANG_FORMAT` and `VALIDATE_CSHARP` too?

Reply:
Yes, you are right, We need c# for the build files and clang in general.